core/vm/eof_instructions.go

```diff
@@ -230,7 +225,7 @@ func opDataLoad(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([
 		stackItem.Clear()
 		scope.Stack.push(&stackItem)
 	} else {
-		data := getData(scope.Contract.Container.data, offset, 32)
+		data := getData(scope.Contract.Container.rawContainer, uint64(scope.Contract.Container.dataOffest)+offset, 32)
```

Comment:
I would like to hide the Container.dataOffset from the EVM, maybe we can add a method to Container, like 
```go
func (c *Container) getData(offset, size int) {
    return getData(c.rawContainer, uint64(c.dataOffest)+offset, size)
}
```